templates/flagger.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: flagger
+  namespace: kubeaddons-flagger
+  labels:
+    kubeaddons.mesosphere.io/name: flagger
+  annotations:
+    sidecar.istio.io/inject: "false"
+    appversion.kubeaddons.mesosphere.io/flagger: "0.19.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.19.0"
+    docs.kubeaddons.mesosphere.io/flagger: "https://docs.flagger.app/"
+    values.chart.helm.kubeaddons.mesosphere.io/flagger: "https://raw.githubusercontent.com/weaveworks/flagger/0.19.0/charts/flagger/values.yaml"
+spec:
+  namespace: kubeaddons-flagger
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: istio
+  chartReference:
+    chart: flagger
+    repo: https://gpaul.github.io/charts/staging
```

Comment:
@GoelDeepak  @shaneutt we should not deploy custom charts from our personal helm repositories

Reply:
that should use mesosphere charts repo

Reply:
I have this feeling that PR was merged before it was intented because the `flagger` charts are still in review

Reply:
Let's revert then so that CI is unblocked.

Reply:
👍

Reply:
:+1: